kubernetes/resource_kubernetes_resource_quota.go

```diff
@@ -55,6 +55,41 @@ func resourceKubernetesResourceQuota() *schema.Resource {
 							Elem:        &schema.Schema{Type: schema.TypeString},
 							Set:         schema.HashString,
 						},
+						"scope_selector": {
+							Type:        schema.TypeList,
+							Description: "A collection of filters like scopes that must match each object tracked by a quota but expressed using ScopeSelectorOperator in combination with possible values. For a resource to match, both scopes AND scopeSelector (if specified in spec), must be matched.",
+							Optional:    true,
+							MaxItems:    1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"match_expression": {
+										Type:        schema.TypeList,
+										Description: "A list of scope selector requirements by scope of the resources.",
+										Optional:    true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"scope_name": {
+													Type:        schema.TypeString,
+													Description: "The name of the scope that the selector applies to.",
+													Required:    true,
+												},
+												"operator": {
```

Comment:
We should add a validator for this to check that it is one of `In`, `NotIn`, `Exists`, `DoesNotExist`.

Reply:
added